frescobaldi_app/widgets/shortcuteditdialog.py

```diff
@@ -31,7 +31,7 @@
 
 import app
 from . import Separator
-from keysequencewidget import KeySequenceWidget
+from . import keysequencewidget
```

Comment:
Does this work? Now we don't have the global `KeySequenceWidget`. You could also do:
```
from .keysequencewidget import KeySequenceWidget
```